circle.yml

```diff
@@ -0,0 +1,20 @@
+machine:
+  node:
+    version: 4.2.1
+  ruby:
+    version: 2.2.3
+
+dependencies:
+  pre:
+    - gem install github_changelog_generator
+    - pip install awscli
+
+test:
+  post:
+    - nvm install 5 && npm test
```

Comment:
Test an additional stable version of node, to preemptively catch issues.  We officially test and support the [LTS](https://nodejs.org/en/blog/release/v4.2.0/).  Thanks to @jhabdas for educating me on `stable` vs `LTS`.